app/controllers/floristry/webtask_controller.rb

```diff
@@ -1,5 +1,5 @@
 module Floristry
-  class WebparticipantController < ::ApplicationController
+  class WebTaskController < ::ApplicationController
```

Comment:
This should be `WebtaskController` and not `WebTaskController`, to fix loading/require issue